### PR TITLE
Provide the correct form for foreach when there is an error.

### PR DESCRIPTION
**Pull request by jpsnover**

### CODE REVIEW

File: src/System.Management.Automation/resources/ParserStrings.resx [hunk 1]
```diff
@@ -420,10 +420,12 @@ Possible matches are</value>
     <value>Missing statement block in switch statement clause.</value>
   </data>
   <data name="MissingForeachExpression" xml:space="preserve">
-    <value>Missing expression in foreach loop.</value>
+    <value>Missing expression in foreach loop.
+The correct form is: foreach ($a in $b) {}</value>
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/resources/ParserStrings.resx]: Would this be slightly more informative?

```none
The correct form is: foreach ($a in $b) { ... }
```

Comment (jpsnover) [reply src/System.Management.Automation/resources/ParserStrings.resx]: Good Q.
I decided to ask users:
https://twitter.com/jsnover/status/914622511412891649

Comment (jpsnover) [reply src/System.Management.Automation/resources/ParserStrings.resx]: Good call!
Twitter is pretty clear on this:
image

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Pull request merged**